### PR TITLE
time zones were backwards!

### PR DESCRIPTION
**Pull request by jason-curtis**
I goofed.

### CODE REVIEW

File: osmo_jupyter/db_access.py [hunk 1]
```diff
@@ -95,8 +95,8 @@ def _get_calculation_details_query(
         SQL query that can be used to get the desired data
     """
 
-    start_timezone_aware_string = _to_utc_string(start_time_local)
-    end_timezone_aware_string = _to_utc_string(end_time_local)
+    start_utc_string = _to_utc_string(start_time_local)
```
Comment (jason-curtis) [inline osmo_jupyter/db_access.py:98]: stuff in this file wasn't really a bug, just incorrect variable names

File: osmo_jupyter/timezone.py [hunk 2]
```diff
@@ -20,5 +20,5 @@ def utc_series_to_local(pandas_timeseries):
     '''
     definitely_a_timeseries = pd.to_datetime(pandas_timeseries)
 
-    localized_timeseries = definitely_a_timeseries.dt.tz_localize(OSMO_HQ_TIMEZONE)
-    return localized_timeseries.dt.tz_convert('UTC').dt.tz_localize(None)
+    localized_timeseries = definitely_a_timeseries.dt.tz_localize('UTC')
```
Comment (jason-curtis) [inline osmo_jupyter/timezone.py:24]: this was previously converting local to UTC, not UTC to local :p

File: osmo_jupyter/timezone.py [hunk 3]
```diff
@@ -20,5 +20,5 @@ def utc_series_to_local(pandas_timeseries):
     '''
     definitely_a_timeseries = pd.to_datetime(pandas_timeseries)
```
Comment (dubchuck) [inline osmo_jupyter/timezone.py]: Fun but maybe something more definite...err...why is it definitely_a_timeseries?

**Review by dubchuck — APPROVED** [review]
One question, otherwise LGTM.

File: osmo_jupyter/timezone.py (hunk 3 above)
Comment (jason-curtis) [reply osmo_jupyter/timezone.py]: because the input may have been a series of strings. I had been meaning to clarify that.

**Pull request merged**